bluemira/codes/plasmod/mapping.py

```diff
@@ -100,6 +100,16 @@ class SOLModel(Model):
     SICCINIO = 1
 
 
+class PLHModel(Model):
+    """
+    L-H transition power scaling model
+
+    6 - Martin
+    """
```

Comment:
For consistency: 

```suggestion
    
    Plasmod variable name: "plh"
    """
```

I was thinking it may be possible to modify `Model` to add a KEY attribute that is only accessible as `Model.KEY` but for now I think this is a good compromise